### PR TITLE
Update gallery, roadmap and community pages

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

File: content/pages/5_roadmap_de.md [hunk 1]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
-Letztes Update der Seite: **23.09.2020**
+Letztes Update der Seite: **10.06.2021**
+
+Für eine komplette Roadmap bis 2024, sehen dieser Dokument: [GMF_2024_Roadmap.pdf](/documents/meetings/2021-06-08/GMF_2024_Roadmap.pdf)
+
+## Version Snook (2.7 LTR)
+
+Nächste Version.
+
+* Entwicklungen: **AnfangSeptember 2021**
```
Comment (maltaesousa) [inline content/pages/5_roadmap_de.md]:
```suggestion
* Entwicklungen: **Anfang September 2021**
```

File: content/pages/5_roadmap_de.md [hunk 2]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+* Geschätztes Freigabedatum: **Summer 2022**
+
+Diese Version habe keine neuen funktionalitäten. Es ist nur eine technische Migration, und das Ziel ist GeoMapFish ohne AngularJS zu bereiten, weil AngularJs ist nicht mehr seit ende 2021 unterstützt.
```
Comment (maltaesousa) [inline content/pages/5_roadmap_de.md]: Shouldn't we talk about the two new functionalities presented today:

- WFS3
- VectorTiles

File: content/pages/5_roadmap_de.md [hunk 3]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
 <br />
 Hier unten finden Sie die Roadmap von GeoMapFish, aber auch die Links zu den Release Notes der verschiedenen Versionen.
 
-Letztes Update der Seite: **23.09.2020**
+Letztes Update der Seite: **10.06.2021**
+
+Für eine komplette Roadmap bis 2024, sehen dieser Dokument: [GMF_2024_Roadmap.pdf](/documents/meetings/2021-06-08/GMF_2024_Roadmap.pdf)
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
Die komplette Roadmap bis 2024 findet sich in diesem Dokument: [GMF_2024_Roadmap.pdf](/documents/meetings/2021-06-08/GMF_2024_Roadmap.pdf)
```

File: content/pages/5_roadmap_de.md [hunk 4]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+
+## Version Snook (2.7 LTR)
+
+Nächste Version.
+
+* Entwicklungen: **Anfang September 2021**
+* Geschätztes Freigabedatum: **Summer 2022**
+
+Diese Version habe keine neuen funktionalitäten. Es ist nur eine technische Migration, und das Ziel ist GeoMapFish ohne AngularJS zu bereiten, weil AngularJs ist nicht mehr seit ende 2021 unterstützt.
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
Diese Version hat keine neuen funktionalitäten. Es ist nur eine technische Migration, und das Ziel ist GeoMapFish vorzubereiten damit es ohne AngularJS funktionnieren kann, da AngularJs seit Ende 2021 nicht mehr unterstützt ist.
```

File: content/pages/5_roadmap_de.md [hunk 5]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+
+### Technische Veränderungen
+
+* Überprüfung neue technische Wahlen mit ein POC.
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
* Überprüfung der neuen technischen Auswahlen mit einem POC.
```

File: content/pages/5_roadmap_de.md [hunk 6]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+* Fusionieren client parts von Ngeo/GMF.
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
* Fusion der client parts von Ngeo/GMF.
```

File: content/pages/5_roadmap_de.md [hunk 7]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+* Auswechseln AngularJs state-management mit RxJS.
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
* Ersatz von AngularJs state-management durch RxJS.
```

File: content/pages/5_roadmap_de.md [hunk 8]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+* Auswechseln AngularJs internationalization stack mit i18next.
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
* Ersatz von AngularJs internationalization stack durch i18next.
```

File: content/pages/5_roadmap_de.md [hunk 9]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+* Integrieren litelement and lithtml zu der Austauch von AngularJS bereiten.
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
* Integration von litelement and lithtml zum Ersatz von AngularJS.
```

File: content/pages/5_roadmap_de.md [hunk 10]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
...
+* Unterstützung Standard Web Components.
```
Comment (kalbermattenm) [inline content/pages/5_roadmap_de.md]:
```suggestion
* Unterstützung von Standard Web Components.
```

**Review by kalbermattenm — APPROVED** [review]
Hope it helps...

File: content/pages/5_roadmap_de.md (hunk 3 above)
Comment (kalbermattenm) [reply content/pages/5_roadmap_de.md]: It's `befindet`... sorry...

File: content/pages/5_roadmap_de.md (hunk 5 above)
Comment (kalbermattenm) [reply content/pages/5_roadmap_de.md]: `Auswahlen` -> `Auswahl`, sorry again...

File: content/pages/5_roadmap_de.md [hunk 11]
```diff
@@ -12,7 +12,29 @@ Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version w
 <br />
 Hier unten finden Sie die Roadmap von GeoMapFish, aber auch die Links zu den Release Notes der verschiedenen Versionen.
 
-Letztes Update der Seite: **23.09.2020**
+Letztes Update der Seite: **10.06.2021**
+
+Die komplette Roadmap bis 2024 findet sich in diesem Dokument: [GMF_2024_Roadmap.pdf](/documents/meetings/2021-06-08/GMF_2024_Roadmap.pdf)
+
```
Comment (pfanguin) [inline content/pages/5_roadmap_de.md]:
```suggestion
Die komplette Roadmap bis 2024 befindet sich in diesem Dokument: [GMF_2024_Roadmap.pdf](/documents/meetings/2021-06-08/GMF_2024_Roadmap.pdf)

```

**Comment by pfanguin** [comment]
Thanks @kalbermattenm for the german review !

**Pull request merged**